tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/CalendarEventReplyPerformer.scala

```diff
@@ -243,6 +246,10 @@ class I18NCalendarEventReplyMessageGenerator(fileSystem: FileSystem, i18nEmlDire
 
   import I18NCalendarEventReplyMessageGenerator._
 
+  private val initCheck: Unit = {
+    Preconditions.checkNotNull(i18nEmlDirectory, s"JMAP CalendarEvent needs a `${I18N_MAIL_TEMPLATE_LOCATION_PROPERTY}` entry".asInstanceOf[Object])
```

Comment:
I didnt notice this at first but why do we force having this parameter defined here? I thought we agreed initially that if it's missing, a default location with default templates would be used?

That threw me a guice injection issue when trying to deploy it on staging

Reply:
> The throwing error when Jmap server startup will help us avoid that

I thought we set a default template in the default configuration of TMail. Or we miss that?

Reply:
Just checked the jib configuration
We already have the default "/root/eml-template" for that 
let me update the code

Reply:
updated & forced push & changed commit message